### PR TITLE
add inputs and function

### PR DESCRIPTION
**Pull request by Murci2022**

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
 		<header>
 			<nav>
-				<Link href="/">Home</Link>
-				<br />
-				<Link href="/createInvoice">Create Invoice</Link>
+				<Wrapper>
+					<Link href="/">Home</Link>
 
-				<br />
+					<Link href="/createInvoice">Create Invoice</Link>
```
Comment (julialoeschel) [inline src/components/Header.jsx:12]: Hey Mate, with the Link, you will need an <a> inside to use with next.js
look here https://nextjs.org/docs/api-reference/next/link

File: src/components/Positioning/InputCard.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+const InputCard = styled.div`
+	display: flex;
+	flex-direction: column;
+	margin: 20px;
+	padding: 5px;
+	border: 1px solid black;
+	list-style: none;
```
Comment (julialoeschel) [inline src/components/Positioning/InputCard.jsx:9]: its a div, are you sure you need a list style here?

File: src/pages/createInvoice.jsx [hunk 3]
```diff
@@ -1,85 +1,185 @@
...
 	const [amount, setAmount] = useState('');
+	const [quantity, setQuantity] = useState('');
+	/* ______________Calculating from quantity and price________ */
+	const subTotal = amount * quantity;
+	const VAT = subTotal * 0.19;
+	const grandTotal = subTotal + VAT;
+
+	/* const subtotal
+	const grandtotal
+	*/
```
Comment (julialoeschel) [inline src/pages/createInvoice.jsx:30]: delete not necessary code here.

File: src/pages/createInvoice.jsx [hunk 4]
```diff
@@ -1,85 +1,185 @@
...
-							</li>
-						</div>
-					</ul>
-					<button onClick={() => setInvoice(false)}>Edit</button>
+					<Wrapper>
+						<h4>Logo</h4>
+						<h4>InvoiceNr.</h4>
+					</Wrapper>
+					<Wrapper>
+						<InputCard>
```
Comment (julialoeschel) [inline src/pages/createInvoice.jsx:41]: Your input Card is a div as far as I can see. `li`shall be used in a list ;)

File: src/pages/createInvoice.jsx [hunk 5]
```diff
@@ -1,85 +1,185 @@
...
+							<li>Adress:{recipientStreet}</li>
+							<li>City:{recipientCity}</li>
+						</InputCard>
+					</Wrapper>
+					<Wrapper>
+						<div>Service:</div>
+
+						<div>price (net):</div>
+
+						<div>QTY:</div>
```
Comment (julialoeschel) [inline src/pages/createInvoice.jsx:58]:
```suggestion
						<div>Service:</div>

						<div>price (net):</div>

						<div>QTY:</div>
```
try to find better semantic elements here

File: src/pages/createInvoice.jsx [hunk 6]
```diff
@@ -1,85 +1,185 @@
...
+					</Wrapper>
+					<Wrapper>
+						<div>-{service}</div>
+						<div>{amount} EUR</div>
+						<div>{quantity}</div>
```
Comment (julialoeschel) [inline src/pages/createInvoice.jsx:63]: same

**Review by julialoeschel — COMMENTED** [review]
good job Mate, 
I found some smaller issues. 
All in all i suggest you to rethink the semanticness of your code once again. 
"do I need that div? " 
"is theis the right heading?" (dont jump from h2 to h4) 
" is this the rigth element?" and so on ;) 

🐜  :octocat:

File: src/components/Positioning/InputCard.jsx (hunk 2 above)
Comment (Murci2022) [reply src/components/Positioning/InputCard.jsx:9]: thank you, i wanted to change this <div> to a <ul> (i have <li> inside) - i correct it now.

File: src/pages/createInvoice.jsx (hunk 4 above)
Comment (Murci2022) [reply src/pages/createInvoice.jsx:41]: This is what I ment before. I changed the div to a ul now:)

**Comment by Murci2022** [comment]
Thank you for your code-review!

**Pull request merged**